app/scripts/components/data-catalog/index.tsx

```diff
@@ -155,153 +182,209 @@ function DataCatalog({ datasets }: DataCatalogProps) {
-                style={{
-                  scrollMarginTop: `${headerHeight + 16}px`
-                }}
-              >
-                <FoldHeadline>
-                  <FoldTitle>Browse</FoldTitle>
-                </FoldHeadline>
-                <BrowseControls
-                  {...controlVars}
-                  taxonomiesOptions={datasetTaxonomies}
-                />
-              </BrowseFoldHeader>
-      
-              <DatasetCount>
-                <span>
-                  Showing{' '}
-                  <Pluralize
-                    singular='dataset'
-                    plural='datasets'
-                    count={displayDatasets.length}
-                    showCount={true}
-                  />{' '}
-                  out of {datasets.length}.
-                </span>
-                {isFiltering && (
-                  <Button forwardedAs={Link} to={DATASETS_PATH} size='small'>
-                    Clear filters <CollecticonXmarkSmall />
-                  </Button>
-                )}
-              </DatasetCount>
-      
-              {displayDatasets.length ? (
-                <CardList>
-                  {displayDatasets.map((d) => {
-                    const topics = getTaxonomy(d, TAXONOMY_TOPICS)?.values;
-                    return (
-                      <li key={d.id}>
-                        <Card
-                          cardType='cover'
-                          overline={
-                            <CardMeta>
-                              <DatasetClassification dataset={d} />
-                              <CardSourcesList
-                                sources={getTaxonomy(d, TAXONOMY_SOURCE)?.values}
-                                rootPath={DATASETS_PATH}
-                                onSourceClick={(id) => {
-                                  onAction(Actions.TAXONOMY, {
-                                    key: TAXONOMY_SOURCE,
-                                    value: id
-                                  });
-                                  browseControlsHeaderRef.current?.scrollIntoView();
-                                }}
-                              />
-                              <VerticalDivider variation='light' />
-                              {/* TODO: Implement modified date: https://github.com/NASA-IMPACT/veda-ui/issues/514 */}
-                              {/* 
-                            <Link
-                              to={`${DATASETS_PATH}?${Actions.SORT_FIELD}=date`}
-                              onClick={(e) => {
-                                e.preventDefault();
-                                onAction(Actions.SORT_FIELD, 'date');
+    <PageMainContent>
+      <LayoutProps
+        title='Data Catalog'
+        description={getString('dataCatalogBanner').other}
+      />
+      <PageHero
+        title='Data Catalog'
+        description={getString('dataCatalogBanner').other}
+      />
+
+      <FeaturedDatasets />
+      <BrowseSection>
```

Comment:
This section leaves no margin on top, makes the general style not complete. (It is not obvious on UI because featured-list has margin at the bottom.) 

<img width="392" alt="Screen Shot 2024-04-25 at 11 39 51 AM" src="https://github.com/NASA-IMPACT/veda-ui/assets/4583806/8fbf1cdb-5ef8-4981-8ba9-b2e7afe63f1d">